engine-cli/src/main/resources/docker-metrics.md

```diff
@@ -1,5 +1,11 @@
 # docker-metrics
 
+```
+NOTE: This feature has been removed and is no longer available.
```

Comment:
We should change this to "... and will be replaced by a simpler mechanism soon." or something similar. Some of our biggest users rely on this and will be expecting a replacement, which we plan to provide.

Reply:
Changed